packages/node-client/src/NodeClient.ts

```diff
@@ -165,6 +165,10 @@ class NodeClient implements INodeClient {
   estimateHandlePaymentGas(
     estimateHandlePaymentTxGasDto: EstimateHandlePaymentTxGasDto
   ): Promise<EstimateGasResponse> {
+    const byPass = process.env.BY_PASS_ESTIMATION
```

Comment:
so this is at env level and not at dto level?
also BY_PASS naming convention looks odd